### PR TITLE
Package mm.0.7.2

### PR DESCRIPTION
**Pull request by toots**
### `mm.0.7.2`
The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)

---
* Homepage: https://github.com/savonet/ocaml-mm
* Source repo: git+https://github.com/savonet/ocaml-mm.git
* Bug tracker: https://github.com/savonet/ocaml-mm/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/mm/mm.0.7.2/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis:
+  "The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "GPL-2.0"
+homepage: "https://github.com/savonet/ocaml-mm"
+bug-reports: "https://github.com/savonet/ocaml-mm/issues"
+depends: [
+  "ocaml" {>= "4.05"}
```
Comment (kit-ty-kate) [inline packages/mm/mm.0.7.2/opam:10]: For the record it failed on OCaml < 4.05 with:
```
#=== ERROR while compiling mm.0.7.2 ===========================================#
# context              2.1.1 | linux/x86_64 | ocaml-base-compiler.4.04.2 | pinned(https://github.com/savonet/ocaml-mm/archive/v0.7.2.tar.gz)
# path                 ~/.opam/4.04/.opam-switch/build/mm.0.7.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p mm -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/mm-2342-b1c423.env
# output-file          ~/.opam/log/mm-2342-b1c423.out
### output ###
#          gcc src/image_rgb.o
# image_rgb.c:71: warning: "Caml_ba_array_val" redefined
#    71 | #define Caml_ba_array_val(v) ((struct caml_ba_array *)Data_custom_val(v))
#       |
# In file included from image_rgb.c:36:
# /home/opam/.opam/4.04/lib/ocaml/caml/bigarray.h:101: note: this is the location of the previous definition
#   101 | #define Caml_ba_array_val(v) ((struct caml_ba_array *) Data_custom_val(v))
#       |
# image_rgb.c: In function 'caml_rgb_color_to_alpha':
# image_rgb.c:1043:10: warning: variable 'd' set but not used [-Wunused-but-set-variable]
#  1043 |   double d;
#       |          ^
# image_rgb.c:1042:22: warning: variable 'aa' set but not used [-Wunused-but-set-variable]
#  1042 |   double rr, gg, bb, aa;
#       |                      ^~
#          gcc src/image_data.o (exit 1)
# (cd _build/default/src && /usr/bin/gcc -std=gnu99 -O2 -fno-strict-aliasing -fwrapv -fcommon -Wall -D_FILE_OFFSET_BITS=64 -D_REENTRANT -fPIC -std=gnu99 -O2 -fno-strict-aliasing -fwrapv -fcommon -Wall -D_FILE_OFFSET_BITS=64 -D_REENTRANT -fPIC -g -I /home/opam/.opam/4.04/lib/ocaml -o image_data.o -c image_data.c)
# image_data.c: In function 'caml_data_aligned':
# image_data.c:31:27: error: 'max_align_t' undeclared (first use in this function)
#    31 |   if (alignment < alignof(max_align_t)) {
#       |                           ^~~~~~~~~~~
# image_data.c:31:27: note: each undeclared identifier is reported only once for each function it appears in
# image_data.c:35:10: warning: implicit declaration of function 'aligned_alloc' [-Wimplicit-function-declaration]
#    35 |   data = aligned_alloc(alignment, len);
#       |          ^~~~~~~~~~~~~
# image_data.c:35:10: warning: incompatible implicit declaration of built-in function 'aligned_alloc'
# image_data.c:15:1: note: include '<stdlib.h>' or provide a declaration of 'aligned_alloc'
#    14 | #include "image_data.h"
#   +++ |+#include <stdlib.h>
#    15 |
```

**Comment by toots** [comment]
Ok thanks. I've just pushed a new release fixing these. Waiting for the assets to be re-generated to get fresh digests.

**Comment by toots** [comment]
All good now!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**